R/derive_locf_records.R

```diff
@@ -115,7 +132,9 @@
 derive_locf_records <- function(dataset,
```

Comment:
Could you add examples for the new arguments?

Reply:
I've added the new arguments for the existing example. Should I add additional examples?

Reply:
We should have at least one example for each value of the `imputation` argument to show the difference between these three ways of imputation.

I would not specify the default value for arguments (`analysis_var` in the example). I think it is better to have a separate example, e.g., specifying `analysis_var = AVALC`.

Reply:
Added additional examples, one for each value of the `imputation` argument, one for `analysis_var = AVALC`.